seed_data/usersData.js

```diff
@@ -26,7 +26,8 @@ var usersData = {
       "_id"       : "58cc4941fc13ae612a10000a",
       "username"  : "Robb Elkson",
       "password"  : "perdonamierastil",
-      "email"     : "summonmonster@erastil.er"
+      "email"     : "summonmonster@erastil.er",
+      "freelancer": ["58cc4942fc13ae612c00004c","58cc4942fc13ae612c000043"]
```

Comment:
They should be correct (one 'in progress', the other 'verified'), but I'm not sure I'm actually using them, because after a merge from Develop a lot of things changed and I had to change a lot of tests. If it's not a problem, I would leave it has it is though.